perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_host.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Steps to bootstrap with gpu docker
+# Install docker
+
+echo "Installing linux utility packages, like, lsb-release, curl..."
+sudo apt-get update
+sudo apt-get install -y \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+echo "Installing docker framework..."
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+
```

Comment:
Also, please add the steps to install the ops-agent and also make sure to run gcsfuse with stack-driver agent.